### ISSUE TITLE
Dockerfile & requirements.txt modification

### ISSUE THREAD

**Issue by koyo922** [faroit/CountNet#1]
- Dockerfile should provide a VOLUME for testing wav files outside the container.
- `protobuf-3.5.0` in `requirements.txt` is not available now, it becomes `protobuf-3.5.0.post1` 

```diff
diff --git a/Dockerfile b/Dockerfile
index 2a75803..3c1098f 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -8,3 +8,5 @@ WORKDIR /app

 RUN pip install --upgrade pip && \
     pip install -r requirements.txt
+
+VOLUME /data
diff --git a/requirements.txt b/requirements.txt
index 9c68fb4..85b1cce 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -11,7 +11,7 @@ llvmlite==0.20.0
 Markdown==2.6.9
 numba==0.35.0
 numpy==1.13.3
-protobuf==3.5.0
+protobuf>=3.5.0
 PyYAML==3.12
 resampy==0.2.0
 scikit-learn==0.19.1
```

**Comment by faroit** [comment]
thanks for spotting this. Feel free to make a pull request

**Comment by faroit** [comment]
addressed in 377c6c1ab62f76644c13285d6276eae68bdf68a0
